### PR TITLE
Remove test access token from the repo

### PR DESCRIPTION
**Pull request by richrines**
- Clean up Gemfile
- Add pry
- Add dotenv

https://trello.com/c/gJ9HmWML

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,35 +1,32 @@
 source 'http://rubygems.org'
 
-gem 'rvm'
```
Comment (jessieay) [inline Gemfile:3]: want to change `.rvmrc` to `.ruby-version` while we're removing this?

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -9,6 +9,7 @@
 # require File.expand_path("../../config/environment", __FILE__)
 require 'webmock/rspec'
 require 'vcr_setup'
+require 'pry'
```
Comment (jessieay) [inline spec/spec_helper.rb:12]: I don't usually have to require pry in `spec_helper` in order for it to work...

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -43,8 +44,4 @@ def expect_id_and_return(id, ret_val)
   lambda { |id| id.to_s.should == id.to_s; ret_val }
 end
 
-def test_access_token
```
Comment (jessieay) [inline spec/spec_helper.rb:46]: LOL

**Comment by jessieay** [comment]
This looks good. I think. :tongue:

File: Gemfile (hunk 1 above)
Comment (richrines) [inline Gemfile:3]: Done, nice call

File: spec/spec_helper.rb (hunk 2 above)
Comment (richrines) [inline spec/spec_helper.rb:12]: Its required as its not being auto-loaded, I'll probably look into ways to change that later on.

**Pull request merged**